.github/workflows/test.yml

```
@@ -56,12 +45,18 @@ jobs:
       # Build the code without test
       - name: Build code (no tests)
         working-directory: .
-        run: mvn -s settings.xml -DskipTests package
+        env:
+          GITHUB_ACTOR: ${{github.actor}}
+          GITHUB_TOKEN: ${{github.token}}
```

```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

github.actor does not apply here unless there is a user triggering it I think, in which case it is set automatically. using the built in token secret and correctly setting the token permissions at the top of the job is the suggested method

---

It does appear that setting `GITHUB_ACTOR` is probably redundant here (the GitHub Maven repos seem to accept an empty user) so both should work but we will know for sure after this change is pushed.
As a note:
1. `github.token` and `secrets.GITHUB_TOKEN` are functionally the same (https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow).
2. `github.actor` is always set (https://github.com/orgs/community/discussions/50250).